examples/hotrod/services/driver/redis.go

```diff
@@ -62,6 +63,8 @@ func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
 	for i := range drivers {
 		drivers[i] = fmt.Sprintf("T7%05dC", rand.Int()%100000)
 	}
+	msg := fmt.Sprintf("FindDriverIDs(%s) = [%s]", location, strings.Join(drivers, ", "))
+	r.logger.For(ctx).Info(msg)
```

Comment:
Zap has a field for arrays: https://github.com/uber-go/zap/blob/master/array.go#L98

```
r.logger.For(ctx).Info("Found drivers", zap.Strings("drivers", drivers))
```

Reply:
Much cleaner thanks. Don't know zap too well.